Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -1248,6 +1249,18 @@ script:
       name: additional_fields
       required: false
       secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'false'
+      description: Flag that indicates whether to set field values using the display
```

Comment:
I didn't understand this. Maybe an example can help.

Reply:
added desc